### PR TITLE
quantity field added to Preference model

### PR DESCRIPTION
**Pull request by AlexisCaffa**
solve issue https://github.com/WhyNotHugo/django-mercadopago/issues/4

### CODE REVIEW

File: django_mercadopago/models.py [hunk 1]
```diff
@@ -189,6 +193,10 @@ class Preference(models.Model):
         max_digits=15,
         decimal_places=2,
     )
+    quantity = models.IntegerField(
```
Comment (WhyNotHugo) [inline django_mercadopago/models.py]: Probably best to restrict this to positive integers.

```suggestion
    quantity = models.`PositiveIntegerField`(
```

File: django_mercadopago/migrations/0017_preference_quantity.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    dependencies = [
+        ('mp', '0016_update_model_meta'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='preference',
+            name='quantity',
+            field=models.IntegerField(
```
Comment (WhyNotHugo) [inline django_mercadopago/migrations/0017_preference_quantity.py]: Probably best to restrict this to positive integers.

```suggestion
            field=models.PositiveIntegerField(
```

File: django_mercadopago/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -58,6 +58,9 @@ msgstr "título"
 msgid "price"
 msgstr "precio"
 
+msgid "quantity"
+msgstr "Cantidad"
```
Comment (WhyNotHugo) [inline django_mercadopago/locale/es/LC_MESSAGES/django.po]: This should keep the same casing:

```suggestion
msgstr "cantidad"
```

**Comment by AlexisCaffa** [comment]
Esta bien el mensaje del chagelog?

**Comment by WhyNotHugo** [comment]
Yup! Thanks!

**Comment by AlexisCaffa** [comment]
:clap: :clap:

**Comment by AlexisCaffa** [comment]
y para que aparezca en el pypi?

**Pull request merged**